modules/s3-articfact/variables.tf

```diff
@@ -0,0 +1,107 @@
+# -----------------------------------------------------------------------------
+# MODULE PARAMETERS
+#
+# These values are expected to be set by the operator when calling the module
+# -----------------------------------------------------------------------------
+
+
+# --------------------------------------------------------------------
+# REQUIRED PARAMETERS
+#
+# These values are required by the module and have no default values
+# --------------------------------------------------------------------
+
+variable "primary_bucket_name" {
+  type        = string
+  description = "The name of the S3 bucket."
+}
+
+variable "replica_bucket_name" {
+  type        = string
+  description = "The name of the S3 bucket."
+}
+
+
+# --------------------------------------------------------------------
+# OPTIONAL PARAMETERS
+#
+# These values are optional and have default values provided
+# --------------------------------------------------------------------
+
+variable "priamry_region" {
+  type        = string
+  description = "The AWS region in which to create the S3 bucket."
+  default     = "us-east-1"
+}
+
+variable "replica_region" {
+  type        = string
+  description = "The AWS region in which to create the S3 bucket."
+  default     = "us-east-2"
+}
+
+variable "pimary_bucket_acl" {
+  type        = string
+  description = "The ACL option to apply to the primary S3 bucket."
+  default     = "private"
+}
+
+variable "replica_bucket_acl" {
+  type        = string
+  description = "The ACL option to apply to the replica S3 bucket."
+  default     = "private"
+}
+
+variable "bucket_versioning_configuration_status" {
```

Comment:
This should be changed to just have `enable_bucket_versioning` and default is `true`